### PR TITLE
Add more structure to tsconfig's

### PR DESCRIPTION
**Pull request by ivy-cst**

### CODE REVIEW

File: config/tsconfig.integration.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "moduleResolution": "Bundler",
```
Comment (ivy-cst) [inline config/tsconfig.integration.json:4]: Set to bunlder according to: https://www.totaltypescript.com/tsconfig-cheat-sheet

File: integrations/standalone/tsconfig.json [hunk 2]
```diff
@@ -1,20 +1,8 @@
...
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
+    "noImplicitOverride": false,
```
Comment (ivy-cst) [inline integrations/standalone/tsconfig.json:5]: Had do change that for the integration, there is a lot of test code which does not declare the override explicitly. May be we should change that?

Comment (ivy-lli) [reply integrations/standalone/tsconfig.json:5]: why is the testcode a problem for the integration?
But yes I'm open for this change

**Review by ivy-lli — APPROVED** [review]
Looks good

File: integrations/standalone/tsconfig.json (hunk 2 above)
Comment (ivy-cst) [reply integrations/standalone/tsconfig.json:5]: In my opinion in the end the noImplicitOverride should be removed here, and so true should be used for this flag as on the tsconfig.base. But at the moment a lot of test code does compile with this flag set to true.

Comment (ivy-lli) [reply integrations/standalone/tsconfig.json:5]: you mean playwright code? or jest code?

**Pull request merged**